tools/static-assets/skel-full/imports/ui/components/info/info.js

```diff
@@ -1,8 +1,10 @@
+import { Links } from '/imports/api/links/links.js';
+import { Meteor } from 'meteor/meteor';
 import './info.html';
-import { Links } from '/imports/api/links/links.js'
-import { Meteor } from 'meteor/meteor'
 
-Template.info.onCreated(function infoOnCreated() {
+/* eslint-disable func-names, prefer-arrow-callback */
```

Comment:
Yep will remove, the eslint config already setup would be ideal for sure, let's go with that.